docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md

```diff
@@ -98,23 +98,43 @@ larger values, the field can be set as a string type instead.
 
 {% info_block infoBox %}
 
-So far the Dynamic Data API supports the following types for the configured fields: boolean, integer, string and decimal.
+So far the Data Exchange API supports the following types for the configured fields: boolean, integer, string and decimal.
 
 {% endinfo_block %}
 
 Let's say you want to have a new endpoint `/dynamic-data/country` to operate with data in `spy_country` table in database.
 
-Based on the provided information, the SQL transaction for interacting with the `spy_country` table through the API request via `dynamic-entity/country` would be as follows:
+In order to configure the endpoint, you need to go to Data Exchange API section in Backoffice and click on "Create Data Exchange API configuration" button.
 
-```sql
-BEGIN;
-INSERT INTO `spy_dynamic_entity_configuration` VALUES ('country', 'spy_country', 1, '{\"identifier\":\"id_country\",\"fields\":[{\"fieldName\":\"id_country\",\"fieldVisibleName\":\"id_country\",\"isEditable\":false,\"isCreatable\":false,\"type\":\"integer\",\"validation\":{\"isRequired\":false}},{\"fieldName\":\"iso2_code\",\"fieldVisibleName\":\"iso2_code\",\"type\":\"string\",\"isEditable\":true,\"isCreatable\":true,\"validation\":{\"isRequired\":true,\"maxLength\":2,\"minLength\":2}},{\"fieldName\":\"iso3_code\",\"fieldVisibleName\":\"iso3_code\",\"type\":\"string\",\"isEditable\":true,\"isCreatable\":true,\"validation\":{\"isRequired\":true,\"maxLength\":3,\"minLength\":3}},{\"fieldName\":\"name\",\"fieldVisibleName\":\"name\",\"type\":\"string\",\"isEditable\":true,\"isCreatable\":true,\"validation\":{\"isRequired\":true,\"maxLength\":255,\"minLength\":1}},{\"fieldName\":\"postal_code_mandatory\",\"fieldVisibleName\":\"postal_code_mandatory\",\"type\":\"boolean\",\"isEditable\":true,\"isCreatable\":true,\"validation\":{\"isRequired\":false}},{\"fieldName\":\"postal_code_regex\",\"isEditable\":\"false\",\"isCreatable\":\"false\",\"fieldVisibleName\":\"postal_code_regex\",\"type\":\"string\",\"validation\":{\"isRequired\":false,\"maxLength\":500,\"minLength\":1}}]}');
-COMMIT;
-```
+[PASTE SCREENSHOT HERE]
+
+Then you need to select a configurable table in the form below. In our case it is `spy_country`:
+
+[PASTE SCREENSHOT HERE]
+
+{% info_block infoBox %}
+
+Note, that you can only select tables that are not excluded from configuring. 
+In order to exclude a table from configuring follow [Data Exchange API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/data-exchange-api-integration.html).
+
+{% endinfo_block %}
+
+After you select a table, you will see a form for configuring the endpoint. Let's fill in the form with the following values:
+
+[PASTE SCREENSHOT HERE]
```

Comment:
<img width="1200" alt="Screenshot 2023-08-21 at 18 01 02" src="https://github.com/spryker/spryker-docs/assets/77281282/60463f36-773f-4c6e-b356-56c9526c0a7b">

Reply:
Why's the `RESOURCE NAME` named `countries` if we are configuring the `/dynamic-data/country` endpoint?

Reply:
Could you also provide a full page screenshot here?

Reply:
> Why's the RESOURCE NAME named countries if we are configuring the /dynamic-data/country endpoint?

`country` was described earlier than the decision to make all resources in the plural was made. Let's better adjust the documentation to use `countries` everywhere

Reply:
<img width="1438" alt="Screenshot 2023-09-15 at 20 34 45" src="https://github.com/spryker/spryker-docs/assets/77281282/cbb1c05e-7c42-40fe-9369-f5174d314026">